heron/executor/src/python/heron_executor.py

```diff
@@ -978,7 +982,9 @@ def start_state_manager_watches(self):
     Receive updates to the packing plan from the statemgrs and update processes as needed.
     """
     statemgr_config = StateMgrConfig()
-    statemgr_config.set_state_locations(configloader.load_state_manager_locations(self.cluster))
+    statemgr_config.set_state_locations(configloader.load_state_manager_locations(
```

Comment:
can you comment on what's going on here?

Reply:
I add this because specifying an override via commandline for the config "heron.statemgr.connection.string" does not get propagated into the heron_executor.  Thus, if I want to specify the location of zookeeper via command line, I cannot.